.github/workflows/releease.yml

```diff
@@ -0,0 +1,122 @@
+name: Release
```

Comment:
mis-spell the file name `releease.yml` -> `release.yml`

Reply:
Thanks! I have fixed it.